tests/services/apiService.test.js

```diff
@@ -40,3 +40,38 @@ describe("fetchQuestCategoryData function", () => {
     expect(result).toEqual(mockResponse);
   });
 });
+
+
+describe("updateUniqueVisitors function", () => {
+  beforeEach(() => {
+    fetch.mockClear();
+  });
+
+  it("should fetch unique visitor count", async () => {
+    const mockData = {
+      res: "unique visitor count",
```

Comment:
the response is this -

```
{
    "res": true
}
```

i would suggest testing the endpoint first and then handling the correct responses for them